app/replica.go

```diff
@@ -92,6 +95,11 @@ checkagain:
 }
 
 func startReplica(c *cli.Context) error {
+	var (
+		volSize            int64
+		isValidSizeDecimal = regexp.MustCompile(`^(\d+(\.\d+)*) ?([kKmMgGtTpP])?[bB]?$`)
```

Comment:
Can we have these expressions outside of function s.t. it can be unit tested.

Reply:
Can you write unit tests for these regexes

Reply:
@AmitKumarDas sure